src/Medulla/Kubernetes/Operator/Entities/V1Alpha1DataEntity.cs

```diff
@@ -0,0 +1,99 @@
+﻿// Licensed to the Medulla Contributors under one or more agreements.
+// The Medulla Contributors licenses this file to you under the Apache 2.0 license.
+// See the LICENSE file in the project root for more information.
+
+using k8s.Models;
+using KubeOps.Operator.Entities;
+using KubeOps.Operator.Entities.Annotations;
+
+namespace DatabaseControllerKubeOps.Controller.Entities;
```

Comment:
Namespace should be `Medulla.Kubernetes.Operator.Entities`

Reply:
Resolved.